src/makeInfinityAdder.js

```diff
@@ -4,7 +4,32 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let totalSum = 0;
+  let currentSum = 0;
```

Comment:
The `totalSum` variable seems to be unnecessary as per the task description. The `currentSum` should be sufficient to keep track of the ongoing sum until the function is called without arguments.